spec/_layouts/default.html

```diff
@@ -1,9 +1,8 @@
 {% assign base = '' %}
 {% assign depth = page.url | split: '/' | size  %}
-{% if    depth <= 1 %}{% assign base = '.' %}
```

Comment:
Is there a reason for deleting the `depth <=1` claues? Is depth never less than two?

Reply:
depth is only 1 , 2  or 3 , and when assigned to  base assigned to '.'   get relative links of "./something.html"  which resulted in bad links Navbar rendering problems.  I didn't investigate this any further just removed the "./"

Reply:
But you also removed the line about depth=1. Was that intentional?